ppml/trusted-big-data-ml/scala/docker-occlum/build_content.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+export DEP_LIBS_DIR="${script_dir}/dep_libs"
+export INITRA_DIR="${script_dir}/init_ra"
+export RATLS_DIR="${script_dir}/../../ra_tls"
+
+function build_ra_tls()
+{
+    rm -rf ${DEP_LIBS_DIR} && mkdir ${DEP_LIBS_DIR}
+    pushd ${RATLS_DIR}
+    bash ./download_and_prepare.sh
```

Comment:
Maybe we can move download _and_prepare into dockerfile

Reply:
This does execute in the dockerfile. https://github.com/intel-analytics/BigDL/pull/4507/files#diff-edda70a3da505fe823ae7ec2480abf25e94c9cf6bbe5f3baa9bc1caea4f34cfdR229

Reply:
Nice design